lib/net/pool.js

```diff
@@ -4342,6 +4461,31 @@ class PoolOptions {
       this.publicPort = options.publicPort;
     }
 
+    if (options.brontideHost != null) {
+      assert(typeof options.host === 'string');
+      const raw = IP.toBuffer(options.brontideHost);
+      this.brontideHost = IP.toString(raw);
+      if (IP.isRoutable(raw))
+        this.brontidePublicHost = this.host;
+    }
+
+    if (options.brontidePort != null) {
+      assert((options.brontidePort & 0xffff) === options.brontidePort);
+      this.brontidePort  = options.brontidePort;
+      this.brontidePublicPort = options.brontidePort;
+    }
+
+    if (options.brontidePublicHost != null) {
+      assert(typeof options.brontidePublicHost === 'string');
+      this.brontidePublicHost = IP.normalize(options.brontidePublicHost);
+    }
+
+    if (options.brontidePublicPort != null) {
+      assert((options.brontidePublicPort & 0xffff)
+        === options.brontidePublicPort);
+      this.brontidePublicPort = options.brontidePublicPort;
+    }
+
     if (options.maxOutbound != null) {
       assert(typeof options.maxOutbound === 'number');
```

Comment:
We should assert that inbound and outbound maximums are integers.

Reply:
I added checks for each of the inbound/outbound integers